.pipelines/cni/pipeline.yaml

```diff
@@ -315,6 +319,17 @@ stages:
       nodeCount: 2
       vmSize: Standard_HC44-16rs
       arch: amd64
+      cni: "cilium"
+  
+  - template: cilium/cilium-overlay-load-test-template.yaml
+    parameters:
+      name: cilium_overlay_ds
+      clusterType: dualstack-byocni-nokubeproxy-up
+      clusterName: "cilium-ds-ov"
+      nodeCount: ${NODE_COUNT_CILIUM}
+      vmSize: ${VM_SIZE_CILIUM}
+      dualstackVersion: ${CILIUM_DUALSTACK_VERSION}
+      cni: "cilium_dualstack"

```

Comment:
Is there a region dependency for cilium dualstack? This would impact cluster creation.

Reply:
We should be able to create clusters for the regions we test with. I was able to get the cluster created in the pipeline a few times last week